server/apps/archive/user_content.py

```diff
@@ -23,7 +23,7 @@ class UserContentResource(Resource):
     datasource = {
         'source': 'archive',
         'aggregations': aggregations,
-        'elastic_filter': {'not': {'exists': {'field': 'task.stage'}}}  # eve-elastic specific filter
+        'elastic_filter': {'not': {'exists': {'field': 'task.desk'}}}  # eve-elastic specific filter
```

Comment:
On personal we have to filter content created by current user and not assigned to any desk.